src/solvers/dgsem_tree/dg_2d_subcell_limiters.jl

```diff
@@ -466,22 +472,40 @@ end
 end
 
 """
-    get_boundary_outer_state(boundary_condition::BoundaryConditionDirichlet,
-                             cache, t, equations, dg, indices...)
+    get_boundary_outer_state(u_inner, t,
+                             boundary_condition::BoundaryConditionDirichlet,
+                             orientation_or_normal, direction,
+                             mesh, equations, dg, cache, indices...)
 For subcell limiting, the calculation of local bounds for non-periodic domains require the boundary
```

Comment:
```suggestion
For subcell limiting, the calculation of local bounds for non-periodic domains requires the boundary
```

Reply:
done in [baf2b30](https://github.com/trixi-framework/Trixi.jl/pull/1954/commits/baf2b307ff106ea3b55eb94f81c1c1abeebf13a5)